airflow/contrib/hooks/aws_sqs_hook.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from airflow.contrib.hooks.aws_hook import AwsHook
+
+
+class SQSHook(AwsHook):
+    """
+    Get the SQS client using boto3 library
+    :return: SQS client
+    :rtype: botocore.client.SQS
+    """
+
+    def get_conn(self):
+        return self.get_client_type('sqs')
```

Comment:
I'm tempted to say that having a subclass for this one method is not needed, and we could just use the AwsHook directly in SQSSensor

Reply:
@ashb - I've added create/send message supporting methods to the hook used by the moto tests.